manifests/a/Anaconda/Anaconda3/2024.02-1/Anaconda.Anaconda3.installer.yaml

```diff
@@ -0,0 +1,33 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS1.CRLF.7-4-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Anaconda.Anaconda3
+PackageVersion: 2024.02-1
+InstallerType: exe
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
+UpgradeBehavior: uninstallPrevious
+Commands:
+- conda
+- pip
+- py
+- python3
+ReleaseDate: 2024-02-26
+AppsAndFeaturesEntries:
+- DisplayName: Anaconda3 2024.02-1 (Python 3.11.7 64-bit)
```

Comment:
```suggestion
- DisplayName: Anaconda3 2024.02-1 (Python 3.11.7 64-bit)
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

Reply:
Tested on win11 VM and seems like zlib.dll should be included with the install and located in C:\ProgramData\anaconda3\Library\lib, in addition it should be available to install via conda: https://anaconda.org/conda-forge/zlib
I agree vcredist should be added, but suggest not including zlib